packages/react-components/react-tabster/src/focus/createFocusOutlineStyle.ts

```diff
@@ -66,10 +66,18 @@ const getFocusOutlineStyles = (options: FocusOutlineStyleOptions): GriffelStyle
  * @returns focus outline styles object for @see makeStyles
  */
 export const createFocusOutlineStyle = ({
+  enableOutline = false,
```

Comment:
we should not add this to `createFocusOutlineStyle`, this utility uses a pseudo element based focus ring which should not be used with browser `outline`. There is currently no way to use the native outline with this utility and I don't think we should introduce a prop to make this possible. If users want to use native outline they should use `createCusomFocusIndicatorStyle`

Reply:
@ling1726 I was thinking it might be useful if someone is using the outline style for something other than the focus ring. E.g. they want to use `outline` instead of border for whatever reason, and use it as an inner outline or otherwise styled in a way that does not conflict with the focus styling.

It's relatively rare, but I've definitely seen and implemented `outline` for non-focus reasons. There's still the out of using `createCustomFocusIndicatorStyle`, but it seemed potentially frustrating to make someone create a custom focus indicator when they just want the default one + the ability to use `outline`

Reply:
the problem here is that the default focus outline is the same thing as what the pseudo element focus outline is actually doing which is to draw an outline around the focused element. We _**should**_ actually make users choose between one or the other in this case, because our default focus indicator _**is an outline**_

Reply:
Here's a bit of a contrived example, but one we might actually want to use:

Toggle button needs an on/off indicator in high contrast mode, so we style an inner outline on the button. Since it's an inner outline, it doesn't conflict with the pseudo-element's focus outline style.

If we were actually using an `outline` style on the button to display focus, I'd style the toggle's on/off inner ring with a pseudo element. But since we're already using one pseudo element for focus, I don't really see a reason to limit the use of `outline` for something like this.

Reply:
Just noting here what we chatted about separately: I'm going to remove `enableOutline` for now, and we can revisit it if we have a concrete use case in the future

Reply:
Also a second note, since I didn't realize initially -- this does also change the public API and behavior of `createFocusOutlineStyle`, since it previously supported `enableOutline` via `createCustomFocusIndicatorStyle`.